### PR TITLE
Interface

### PR DESCRIPTION
**Pull request by YahyaAzzam**

### CODE REVIEW

File: src/main/java/Driver.java [hunk 1]
```diff
@@ -2,13 +2,39 @@
...
+
+    }
 
+    @Override
+    public JSObject[] getRow(final String databaseName, final String tableName, final String value) {
+        return null;
+    }
+
+    @Override
+    public void deleteRow(final String databaseName, final String tableName, final String value) {
```
Comment (mohamedabdelsalam) [inline src/main/java/Driver.java]: Should return true if received succeeded

File: src/main/java/Driver.java [hunk 2]
```diff
@@ -2,13 +2,39 @@
...
+
+    }
+
+    @Override
+    public void clearDatabase(final String databaseName) {
```
Comment (mohamedabdelsalam) [inline src/main/java/Driver.java]: Should return boolewn

File: src/main/java/Driver.java [hunk 3]
```diff
@@ -2,13 +2,39 @@
 
 //import com.google.common.base.Strings;
 
+import netscape.javascript.JSObject;
```
Comment (mohamedabdelsalam) [inline src/main/java/Driver.java]: Use this for json

Comment (YahyaAzzam) [reply src/main/java/Driver.java]: I don't understand

**Review by mohamedabdelsalam — APPROVED** [review]

**Pull request merged**